lean4-infoview/src/infoview/rpcInterface.ts

```diff
@@ -65,6 +65,8 @@ export async function InteractiveDiagnostics_infoToInteractive(rs: RpcSessions,
 }
 
 export interface InteractiveHypothesis {
+    isInstance: boolean,
+    isType: boolean,
```

Comment:
```suggestion
    isInstance?: boolean,
    isType?: boolean,
```
Since this property is not present in older Lean versions.

Reply:
thanks